src/ng/http.js

```diff
@@ -909,7 +914,7 @@ function $HttpProvider() {
   var data = element(by.binding('data'));
   var fetchBtn = element(by.id('fetchbtn'));
   var sampleGetBtn = element(by.id('samplegetbtn'));
-  var sampleJsonpBtn = element(by.id('samplejsonpbtn'));
+//  var sampleJsonpBtn = element(by.id('samplejsonpbtn'));
```

Comment:
To avoid confusion, we could move this line inside the commented out test (where there is a comment with more context.